terraform/api_gateway.tf

```diff
@@ -0,0 +1,9 @@
+module "api_gateway" {
```

Comment:
Please remove the entire module. either create a simple apigateway (no module or domain or cert) or just create a S3 bucket. The goal here is to just create a aws resource not a functional one.